roles/candlepin/tasks/certs.yml

```diff
@@ -10,6 +10,17 @@
     other_certificates: "{{ candlepin_ca_certificate }}"
     state: present
 
+- name: Generate truststore
+  community.crypto.openssl_pkcs12:
+    action: export
+    passphrase: "{{ candlepin_keystore_password }}"
+    path: "/root/candlepin.truststore"
+    friendly_name: 'artemis-client'
+    privatekey_path: "{{ candlepin_client_key }}"
+    certificate_path: "{{ candlepin_client_certificate }}"
+    other_certificates: "{{ candlepin_ca_certificate }}"
+    state: present
```

Comment:
This is not really a truststore, more a keystore.
But when I remove `privatekey_path`, Ansible generated an empty keystore :thinking: 

But I need a store that has `artemis-client` in it.

Reply:
Yeah will do.

Reply:
https://github.com/ehelms/foreman-quadlet/pull/8